src/less/signals/base/signals.less

```diff
@@ -0,0 +1,37 @@
+@import "../../mixins/utility/utility-mixins.less";
+
+.signal {
+    align-items: center;
+    border: 1px solid;
+    border-radius: 8px;
+    box-sizing: border-box;
+    display: flex;
+    font-family: MarketSans, sans-serif;
+    font-size: 10px;
+    font-weight: bold;
+    letter-spacing: 0.5px;
+    line-height: 12px;
```

Comment:
Line-height generally causes problems and I'm not sure we really need it. If we do, it should be a non-pixel unit. Try seeing what happens when text-size is increased.